ios/FluentUI/Tab Bar/SideTabBar.swift

```diff
@@ -175,21 +177,29 @@ open class SideTabBar: UIView {
         }
 
         if let avatarView = avatarView {
+            let topSafeConstraint = avatarView.topAnchor.constraint(equalTo: safeAreaLayoutGuide.topAnchor, constant: Constants.avatarViewSafeTopSpacing)
```

Comment:
can we comment why you are doing this? i am not sure it is obvious at first sight.